sw/device/tests/sim_dv/flash_escalation_reset_test.c

```diff
@@ -131,24 +202,32 @@ static void alert_handler_config(void) {
 
   dif_alert_handler_escalation_phase_t esc_phases[] = {
       {.phase = kDifAlertHandlerClassStatePhase0,
-       .signal = 0,
+       .signal = 0xFFFFFFFF,  // do not trigger any signal, just wait.
        .duration_cycles =
-           alert_handler_testutils_get_cycles_from_us(kEscalationPhase0Micros) *
-           alert_handler_testutils_cycle_rescaling_factor()},
+           alert_handler_testutils_get_cycles_from_us(kEscalationPhase0Micros)},
       {.phase = kDifAlertHandlerClassStatePhase1,
-       .signal = 3,
+       .signal = 0,  // NMI
+       .duration_cycles =
+           alert_handler_testutils_get_cycles_from_us(kEscalationPhase1Micros)},
+      {.phase = kDifAlertHandlerClassStatePhase2,
+       .signal = 1,  // lc_escalate_en
        .duration_cycles =
-           alert_handler_testutils_get_cycles_from_us(kEscalationPhase1Micros) *
-           alert_handler_testutils_cycle_rescaling_factor()}};
-
-  uint32_t deadline_cycles =
-      alert_handler_testutils_get_cycles_from_us(kEscalationStartMicros) *
-      alert_handler_testutils_cycle_rescaling_factor();
-  LOG_INFO("Configuring class A with %d cycles and %d occurrences",
-           deadline_cycles, UINT16_MAX);
+           alert_handler_testutils_get_cycles_from_us(kEscalationPhase2Micros)},
+      {.phase = kDifAlertHandlerClassStatePhase3,
+       .signal = 3,  // reset
+       .duration_cycles = alert_handler_testutils_get_cycles_from_us(
+           kEscalationPhase3Micros)}};
+
+  // This test does not leverage the IRQ timeout feature of the alert
+  // handler, hence deadline_cycles is set to zero. Rather, it triggers
+  // escalation right away if an alert event is seen, hence threshold = 0;
+  uint32_t deadline_cycles = 0;
+  uint32_t threshold = 0;
+
   dif_alert_handler_class_config_t class_config[] = {{
       .auto_lock_accumulation_counter = kDifToggleDisabled,
-      .accumulator_threshold = UINT16_MAX,
+      //      .accumulator_threshold = UINT16_MAX,
```

Comment:
nit: can you remove this commented out line?

Reply:
yes